mhkit/tests/utils/test_cache.py

```diff
@@ -114,8 +118,18 @@ def test_handle_caching_retrieves_data(self):
         Asserts:
         - The retrieved data matches the original sample DataFrame.
         """
-        handle_caching(self.hash_params, self.cache_dir, data=self.data)
-        retrieved_data, _, _ = handle_caching(self.hash_params, self.cache_dir)
+        handle_caching(
+            self.hash_params,
+            self.cache_dir,
+            cache_content={"data": self.data, "metadata": None, "write_json": None},
+        )
+
+        retrieved_data, _, _ = handle_caching(
+            self.hash_params,
+            self.cache_dir,
+            cache_content={"data": None, "metadata": None, "write_json": None},
+        )
+
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args